.travis.yml

```diff
@@ -1,21 +1,25 @@
 language: python
 matrix:
-  - python: '2.7'
-  - python: '3.5'
-  - python: '3.6'
-  - python: '3.7'
-    dist: xenial
-env:
-  - CONFIG=buildout.cfg
+    include:
+        - python: "3.5"
+          env: TOXENV=py35
+        - python: "3.6"
+          env: TOXENV=py36
+        - python: "3.7"
+          env: TOXENV=py37
+
 install:
-  - pip install -U pip setuptools zc.buildout
-  - buildout parts=test -c $CONFIG
-before_script:
-  - export TZ=US/Eastern
```

Comment:
Might this change be related to the failing tests?  Just spitballing.

Reply:
Restoring the `before_script` and the timezone export doesn't seem to change anything.

The last time Travis tests ran successfully was 3 years ago(!). The symptoms are always the same. Tests run fine until it hits tests run under class `zc.zrs.tests.ZEOHexClientHexTests`. These use a special storage "Hex storage". The tests run super slow on Travis, they show test failures and eventually just hit the 50 minute execution limit because they are so slow. (I personally don't even know what that Hex Storage is for and if anyone actually uses that).

Reply:
HexStorage is for testing storage wrapper. An example wrapper is `zc.zlibstorage` which compresses data; wrappers are supposed to work on top of any storage. HexStorage is a simple wrapper that transforms pickle data from arbitrary bytes to hexadecimal representation for storage and back again for loading. So I guess those tests are to be sure that wrappers work.